subsys/bluetooth/shell/bt.c

```diff
@@ -189,6 +189,24 @@ static const char *enabled2str(bool enabled)
 
 #endif /* CONFIG_BT_TRANSMIT_POWER_CONTROL */
 
+#if defined(CONFIG_BT_PATH_LOSS_MONITORING)
+static const char *plm_report_zone2str(enum bt_conn_le_path_loss_zone zone_entered)
```

Comment:
In Bluetooth in Zephyr it seems more common to either do 
```suggestion
static const char *plm_report_zone_str(enum bt_conn_le_path_loss_zone zone_entered)
```
or
```suggestion
static const char *plm_report_zone_to_str(enum bt_conn_le_path_loss_zone zone_entered)
```